### ISSUE TITLE
fix COUCH_NODE_NAME

### ISSUE THREAD

**Issue by Hareet** [medic/medic-os#46]
Variable COUCH_NODE_NAME is located in many places and hardcoded to the old value of `couchdb@localhost`. This needs to be changed and an image rebuilt to have `couchdb@127.0.0.1`

**Comment by SCdF** [comment]
@Hareet as this blocks https://github.com/medic/medic/issues/5499 can you explain how to work around this to set it in a live instance?

**Comment by Hareet** [comment]
Workaround:
in `/srv/scripts/medic-api/run/medic-api` && `/srv/scripts/medic-sentinel/run/medic-sentinel`
fix `COUCH_NODE_NAME='couchdb@127.0.0.1'` 
and run:
```
/boot/svc-restart medic-api
/boot/svc-restart medic-sentinel
```

fwiw: In fresh installs, there's differing scripts that run and set the variable elsewhere. This will be globally fixed.

**Comment by Hareet** [comment]
Fixed in image: `medicmobile/medic-os:cht-3.7.0-rc.1`
